### PR TITLE
walk: init at 1.5.2

### PR DESCRIPTION
**Pull request by surfaceflinger**
## Description of changes

 - Upstream [changed name ](https://github.com/antonmedv/walk/commit/72e63865a02c5c8f75e3f2196279714323fafc4f)from `llama` to `walk` so I changed name of the derivation and created an alias
 - Bumped version to 1.5.2
 - Added myself to maintainers

pinging current maintainer @portothree

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by lelgenio** [comment]
Result of `nixpkgs-review pr 250847` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>walk</li>
  </ul>
</details>

**Comment by portothree** [comment]
Thank you!

**Review by portothree — APPROVED** [review]

File: pkgs/applications/file-managers/walk/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  };
+
+  vendorHash = "sha256-EYwfoTVcgV12xF/cv9O6QgXq9Gtc9qK9EmZNjXS4kC8=";
+
+  meta = with lib; {
+    description = "Terminal file manager";
+    homepage = "https://github.com/antonmedv/walk";
+    license = licenses.mit;
+    maintainers = with maintainers; [ portothree surfaceflinger ];
+  };
```
Comment (antonmosich) [inline pkgs/applications/file-managers/walk/default.nix:21]: Consider adding the meta.mainProgram attribute here. Especially when using the llama alias with lib.getExe that would make a difference.

Comment (surfaceflinger) [reply pkgs/applications/file-managers/walk/default.nix:21]: Do you think if this should be set explicitly to "walk" or reuse pname?

**Comment by antonmosich** [comment]
I would set it to `walk` explicitly.

File: pkgs/applications/file-managers/walk/default.nix (hunk 1 above)
Comment (AndersonTorres) [reply pkgs/applications/file-managers/walk/default.nix:21]: Do not parameterize `pname`.

**Review by AndersonTorres — COMMENTED** [review]
I recommend a bit of bureaucracy to keep the tree "always-buildable":

1. A commit adding `walk: init at 1.5.2`
2. The next one removing and aliasing llama

File: pkgs/applications/file-managers/walk/default.nix [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+buildGoModule rec {
+  pname = "walk";
+  version = "1.5.2";
+
+  src = fetchFromGitHub {
+    owner = "antonmedv";
+    repo = "walk";
+    rev = "v${version}";
+    sha256 = "sha256-lcXNGmDCXq73gAWFKHHsIb578b1EhznYaGC0myFQym8=";
```
[the remark this answers is not in the document]
Comment (surfaceflinger) [reply pkgs/applications/file-managers/walk/default.nix]: done!

**Review by portothree — APPROVED** [review]

**Pull request merged**